docs/examples/Transformer Examples.ipynb

```diff
@@ -533,16 +527,1066 @@
+       "   id            u\n",
+       "0   2  9999.994657\n",
+       "1   4   393.881868\n",
+       "2   6   334.529465"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "------tap regulator result------\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>id</th>\n",
+       "      <th>energized</th>\n",
+       "      <th>tap_pos</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>8</td>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   id  energized  tap_pos\n",
+       "0   8          1        1"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# one-time power flow calculation with automatic tap changing\n",
+    "output_data5 = model5.calculate_power_flow(tap_changing_strategy=TapChangingStrategy.min_voltage_tap)\n",
+    "\n",
+    "print(\"------node result------\")\n",
+    "display(pd.DataFrame(output_data5[\"node\"])[[\"id\", \"u\"]])\n",
+    "\n",
+    "print(\"\\n------tap regulator result------\")\n",
+    "display(pd.DataFrame(output_data5[\"transformer_tap_regulator\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "01a473bd",
+   "metadata": {},
+   "source": [
+    "### Automatic tap changing with line drop compensation\n",
+    "\n",
+    "In the above examples, the voltage at the low-voltage side of the transformer is controlled. However, the load is still connected to a node with a voltage lower than 390 V.\n",
+    "\n",
+    "To regulate the node that the lode is connected to, the voltage needs to be compensated for the voltage drop over the line.\n",
+    "\n",
+    "```txt\n",
+    "                   (tap_side)  (control side)\n",
+    "source_1 --- node_2 --- transformer_3 --- node_4 --- line_5 --- node_6 --- load_7\n",
+    "                        |                 |                         :\n",
+    "                        regulator_8 <-----/ <......................./ (line drop compensation)\n",
+    "                                     (control voltage) \n",
+    "```"
```

Comment:
Resolve without changes:

I wish there was a draw.io like tool for power grid. What do you think about changing these to hand drawn image svgs from whiteboard instead?

Reply:
we have a couple options: we can use GraphViz or we can get some input from the team that's working on the data science toolkit

Reply:
let's talk about it when discussing the improvmenets this summer